src/script/components/Giphy/GifImage.test.tsx

```diff
@@ -21,8 +21,8 @@ import {render, fireEvent} from '@testing-library/react';
 
 import GifImage from 'Components/Giphy/GifImage';
 
-const staticImage = 'https://unsplash.it/500/300';
-const hoveredImage = 'https://unsplash.it/500/400';
+const staticImage = 'https://mock.image/500/300';
```

Comment:
Instead of `mock.image` please use `image.invalid`. At some point in the future the `.image` top-level-domain (TLD) could be registered. Whereas the `.invalid` TLD is permanently reserved.

Reply:
That's good call :) 
Here it's no big deal as we are only testing component rendering, there are no http request made at any point in our unit tests. 
But that's valid suggestion 👍

Reply:
Oh good point, did't think about it 😄